react_projects/rick-and-morty-web/src/components/tableContent/TableContent.js

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react';
+import RawItem from '../rawItem/RawItem';
+
+import './TableContent.css'
+import Pagination from '../pagination/Pagination';
+import FilterBar from '../filterBar/filterBar';
+import Loader from '../Loader/Loader';
+import useDebounce from '../../hooks/useDebounce';
+
+const TableContent = () => {
+
+  const [data, setData] = useState(null);
+  const [searchInput, setSearchInput] = useState('')
+  const debouncedSearchValue = useDebounce(searchInput)
+  const [pageNumber, setPageNumber] = useState(1);
+
+  useEffect(() => {
+    fetch(`https://rickandmortyapi.com/api/character?page=${pageNumber}&name=${debouncedSearchValue}`)
```

Comment:
Some other ways to fetch GET with query params https://stackoverflow.com/questions/35038857/setting-query-string-using-fetch-get-request